npm/ng-packs/packages/account/src/lib/guards/authentication-flow.guard.ts

```diff
@@ -12,3 +15,12 @@ export class AuthenticationFlowGuard implements IAbpGuard {
     return false;
   }
 }
+
+export const authenticationFlowGuard = () => {
```

Comment:
Missing method sign `CanActivateFn` if method sings not implemented please normalize all guards/resolves